.circleci/config.yml

```
@@ -157,6 +157,16 @@ jobs:
                 command: bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE}
                 name: '[FL] Build Android'
                 working_directory: android
+            - run:
+                command: |
+                    mkdir -p sourcemaps/
+                    cp android/app/build/generated/sourcemaps/react/release/index.android.bundle.map sourcemaps/
+                    cp android/app/build/generated/assets/react/release/index.android.bundle sourcemaps/
```

```suggestion
                    cp android/app/build/generated/sourcemaps/react/release/index.android.bundle* sourcemaps/
```

more similar to below

---

Sorry i don't get it. Once the path is `android/app/build/generated/**assets**/...` and once it is `android/app/build/generated/**sourcemaps**/..`. So i cannot just replace line 163 and line 164 with the solution you suggested, can I?

---

oh sorry, did not see it!